Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -39,7 +39,11 @@ public class BTShopperInsightsClient {
             email: request.email,
             phone: request.phone
         )
-        
+
+        if apiClient.authorization.type != .clientToken {
```

Comment:
Take it or leave it for readability - move this above the `postParameters` construction